memory_model/src/guest_memory.rs

```diff
@@ -139,6 +139,15 @@ impl GuestMemory {
         self.regions.len()
     }
 
+    /// Returns the size region identified by index
+    pub fn region_size(&self, index: usize) -> usize {
+        if index >= self.regions.len() {
```

Comment:
In this case I think it's better to return an error.

Reply:
It is returning an `Result<usize>` in the latest commit, I have messed up with commit orders / squashing